src/database/migrations/20251020081719-add-orgEntity-type.js

```diff
@@ -28,8 +28,8 @@ module.exports = {
 					updated_by: 0,
 					allow_filtering: false,
 					organization_id: defaultOrgId,
-					organization_code: defaultOrgCode,
-					tenant_code: defaultTenantCode,
+					//organization_code: defaultOrgCode,
+					//tenant_code: defaultTenantCode,
```

Comment:
Why is this commented out?

Reply:
<img width="939" height="475" alt="image" src="https://github.com/user-attachments/assets/7d6967fe-6608-4695-aa9c-b3f2a8a242a0" />
The migration fail if I do not comment this. 
The org and tenant anyways added in the 20251212250000-complete-tenant-code-migration.js script eventually.